synapseAnnotations/data/neuro.json

```diff
@@ -16,113 +16,7 @@
     "description": "", 
```

Comment:
Will you add a description of what this key represents? Without the enumvalues it's not self-explanatory.

Reply:
We will be using 'group' as a key in the PhaseII PEC schema. Values will be 'institution-PI'